core/views/browse.py

```diff
@@ -23,6 +23,7 @@
 from chronam.core.utils.url import unpack_url_path
 from chronam.core import models, index
 from chronam.core.rdf import title_to_graph, issue_to_graph, page_to_graph
+from chronam.core.index import get_page_text
```

Comment:
Nothing wrong with this import but it would be nice to have them sorted using something like `isort` for readability.

Reply:
I agree, but can it be done later, maybe as a part of a different ticket?

Reply:
Why not add a commit before merging this branch? It should be quick.

Reply:
I applied isort

Reply:
Thanks!